### PR TITLE
storage/txn: tiny cleanup

### PR DESCRIPTION
**Pull request by BusyJay**
@ngaut @siddontang @disksing @zhangjinpeng1987 PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: src/storage/txn/scheduler.rs [hunk 1]
```diff
@@ -656,10 +656,11 @@ impl Scheduler {
                                  pr: ProcessResult,
                                  to_be_write: Vec<Modify>) {
         SCHED_STAGE_COUNTER_VEC.with_label_values(&[self.get_ctx_tag(cid), "write"]).inc();
-        if let Err(e) = {
-            let engine_cb = make_engine_cb(cid, pr, self.schedch.clone());
-            self.engine.async_write(extract_ctx(&cmd), to_be_write, engine_cb)
-        } {
+        if to_be_write.is_empty() {
+            return self.on_write_finished(cid, pr, Ok(()));
```
Comment (siddontang) [inline src/storage/txn/scheduler.rs]: any test to cover this?

Comment (BusyJay) [inline src/storage/txn/scheduler.rs]: Current tests cover it already.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**